devices/gledopto.js

```diff
@@ -466,6 +466,20 @@ module.exports = [
         description: 'Zigbee 12W Downlight RGB+CCT (pro)',
         extend: gledoptoExtend.light_onoff_brightness_colortemp_color(),
     },
+    {
+        zigbeeModel: ['GL-D-006P'],
+        model: 'GL-D-006P',
+        vendor: 'Gledopto',
+        description: 'Zigbee 6W Anti-glare Downlight RGB+CCT (pro)',
+        extend: extend.light_onoff_brightness_colortemp_color({colorTempRange: [158, 495]}),
```

Comment:
Did you use `extend` instead of `gledoptoExtend` on purpose?

Reply:
Yes, not sure how I managed to get it working last week, but this week it definitely don't work. I have the following error while starting Z2M:
```Using '/app/data' as data directory
(node:7) UnhandledPromiseRejectionWarning: ReferenceError: gledoptoExtend is not defined
(Use `node --trace-warnings ...` to show where the warning was created)
(node:7) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:7) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

This is my first integration, so I may be doing something wrong...

For reference, here are my edited and working files:
/data/GLD006P.js
```const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const e = exposes.presets;
const ea = exposes.access;

const definition = {
    zigbeeModel: ['GL-D-006P'],
    model: 'GL-D-006P',
    vendor: 'Gledopto',
    description: 'Zigbee 6W Anti-glare Downlight RGB+CCT (pro)',
    extend: extend.light_onoff_brightness_colortemp_color({colorTempRange: [158, 495]}),
};

module.exports = definition;```

/app/configuration.yaml
```homeassistant: false
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://localhost:1884
  user: xxxx
  password: xxxx
serial:
  port: /dev/ttyACM0
frontend:
  port: 8080
map_options:
  graphviz:
    colors:
      fill:
        enddevice: '#fff8ce'
        coordinator: '#e04e5d'
        router: '#4ea3e0'
      font:
        coordinator: '#ffffff'
        router: '#ffffff'
        enddevice: '#000000'
      line:
        active: '#009900'
        inactive: '#994444'
devices:
  '0x84fd27fffebb92c4':
    friendly_name: '0x84fd27fffebb92c4'
  '0x84fd27fffe931816':
    friendly_name: '0x84fd27fffe931816'
advanced:
  log_level: debug
external_converters:
  - GLD006P.js```

Reply:
This is because the `gledoptoExtend` is defined only in this file, I've changed it to `gledoptoExtend`, thanks!